src/flow.jl

```diff
@@ -59,6 +187,8 @@ function update(
     # use fixed alpha value based on 0.5 * h_bankfull
     @. sf.α = sf.alpha_term * pow(sf.width + sf.h_bankfull, sf.alpha_pow)
 
+    @. sf.qlat .= sf.inwater ./ sf.dl
```

Comment:
If you use `@.` you don't need to use . operators for broadcasting as well.

Reply:
did remove the . operators.